pcdsdevices/tests/test_lens_sets/test

```diff
@@ -1 +1 @@
-[2, 0.0002, 4, 0.0005]
\ No newline at end of file
+[2, 0.0002, 4, 0.0005]
```

Comment:
I'm also confused as to why the build is happier with an `__init__.py` added into this directory- this directory didn't have any python files in it prior

Reply:
A directory without an `__init__.py` is an implicit namespace package. The presence of `__init__.py` in there disables that (see https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#native-namespace-packages).

I don't think the implicit namespace packages have a place for us, and the addition of `__init__.py` makes it obvious what should be importable (for setuptools and for us, I think). Maybe my perspective is dated, though.

Reply:
The question is: there isn't anything to import here, it's just files that include data, so why should we want/need to add an importable file?

Reply:
For myself, I've always just included `__init__.py` so I can be sure that regardless of the `setup.py` / `MANIFEST` / `setup.cfg` / etc settings, the directory I intended to include will be available in the distributed package. 

It's possible that setting `namespaces` to `True` and removing extraneous `__init__.py` files could be fine - I'm just not confident about it. The best way of doing things is likely buried somewhere in here or the link above: https://setuptools.pypa.io/en/latest/userguide/package_discovery.html